src/frontend/sharedComponents/DeviceCard.tsx

```diff
@@ -44,30 +54,60 @@ export const DeviceCard = ({
       throw new ExhaustivenessError(deviceConnectionStatus);
   }
 
+  const {navigate} = useNavigationFromRoot();
+  const {formatMessage} = useIntl();
+
   return (
     <TouchableOpacity
       disabled={!onPress || isDisconnected}
       onPress={() => (onPress ? onPress() : {})}
-      style={[styles.container, style]}>
-      <DeviceNameWithIcon
-        name={name}
-        deviceConnectionStatus={deviceConnectionStatus}
-        thisDevice={thisDevice}
-        deviceType={deviceType}
-        deviceId={deviceId}
-        iconSize={75}
-      />
+      style={[style]}>
+      <View style={styles.container}>
+        <DeviceNameWithIcon
+          style={{flexShrink: 1}}
+          name={name}
+          deviceConnectionStatus={deviceConnectionStatus}
+          thisDevice={thisDevice}
+          deviceType={deviceType}
+          deviceId={deviceId}
+          iconSize={75}
+        />
+        <View style={{alignSelf: 'flex-start'}}>
+          {dateAdded && (
+            <Text style={{fontSize: 12, marginBottom: 10}}>
+              <FormattedDate
+                value={new Date(dateAdded)}
```

Comment:
I don't think this needs to be converted into a date first

```suggestion
                value={dateAdded}
```

doing so doesn't play nicely with React either because you're creating a new instance on each render

Reply:
https://github.com/digidem/comapeo-mobile/pull/674/commits/7704b2a2aa999a780a5aa0c368dc5b197b353e89